packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -300,23 +300,32 @@ class WebdevFixture {
     final Completer<String> hasUrl = Completer<String>();
 
     _toLines(process.stderr).listen((String line) {
+      if (verbose) {
+        print('pub run build_runner serve • ${process.pid}'
+            ' • STDERR • ${line.trim()}');
+      }
+
       final err = 'error starting webdev: $line';
       if (!hasUrl.isCompleted) {
         hasUrl.completeError(err);
       } else {
-        print(err);
+        print('Ignoring stderr output because already completed');
```

Comment:
do we also want to include `err` somewhere in this message?

Reply:
Er, yeah! I moved it further up, but it was only in verbose mode - so in non-verbose mode it won't show up. I'll fix, thanks!

Reply:
Pushed a fix